scripts/install_chipsec.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 lsmod | grep chipsec 1>/dev/null 2>&1 && echo "Chipsec already insalled :)" && exit
-python3 -m pip install -U pip && python3 -m pip install -U setuptools
+(python3 -m pip install -U pip && python3 -m pip install -U setuptools) || true
```

Comment:
What does this do?

Reply:
Some python environments does not allow installing setup tools, so if it fails I return true from this line.